.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+### Summary
+
+Add a summary for the PR
+
+### Checklist
+
+- [ ] Tested the code and it does not break anything
+- [ ] `npm run format:fix`
+- [ ] Commits / PR title are easy for others to follow :smiley:
+- [ ] Include a screenshot if it's for a view :sunglasses:
```

Comment:
I think this I did this correctly, it should add a default PR summary and checklist you can use when you create a PR. Unfortunately, I dont think the smiley face or the sunglasses will work 😭 

@CleveyP @CaiminRO @ahuda620